Makefile

```diff
@@ -74,8 +74,7 @@ install: manifests
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
```

Comment:
Update the comment to help the developers to customise the flags, as discussed. Also, can the developer provide any other KUBECONFIG than the one mentioned here? if so, mention that in comment. if no, please add the provision.

Reply:
Fixed.